examples/text_box.rs

```diff
@@ -13,35 +15,43 @@ use kayak_ui::widgets::{App, OnChange, TextBox, Window};
 
 #[widget]
 fn TextBoxExample(context: &mut KayakContext) {
-    let value = context
-        .create_state("I started with a value!".to_string())
-        .unwrap();
-    let value2 = context.create_state("".to_string()).unwrap();
+    let (value, set_value, _) = use_state!("I started with a value!".to_string());
+    let (empty_value, set_empty_value, _) = use_state!("".to_string());
+    let (red_value, set_red_value, _) = use_state!("This text is red".to_string());
```

Comment:
With this change could you remove the `Bound` and `MutableBound` imports? They're automatically added by the `use_state!` macro.

Reply:
Yep, sorry, rust-analyzer is going a bit crazy with me constantly changing branches and recompiling

Reply:
No worries!